src/amo/components/Addon/index.js

```diff
@@ -266,6 +298,67 @@ export class AddonBase extends React.Component {
     /* eslint-enable react/no-danger */
   }
 
+  renderMoreAddonsByAuthors = () => {
+    const { addon, addonsByAuthors, i18n } = this.props;
+
+    if (!addon || !addonsByAuthors || addonsByAuthors.length === 0) {
+      return null;
+    }
+
+    const addonType = addon.type;
+    const authorNames = addon.authors.map((author) => author.name);
+
+    let header;
+    switch (addonType) {
+      case ADDON_TYPE_EXTENSION:
+        header = i18n.ngettext(
+          i18n.sprintf(
+            i18n.gettext('More extensions by %(author)s'),
+            { author: authorNames[0] }
+          ),
+          i18n.gettext('More extensions by these authors'),
+          authorNames.length
+        );
+        break;
+
+      case ADDON_TYPE_THEME:
+        header = i18n.ngettext(
+          i18n.sprintf(
+            i18n.gettext('More themes by %(author)s'),
+            { author: authorNames[0] }
+          ),
+          i18n.gettext('More themes by these authors'),
+          authorNames.length
+        );
+        break;
+
+      default:
+        header = i18n.ngettext(
+          i18n.sprintf(
+            i18n.gettext('More add-ons by %(author)s'),
+            { author: authorNames[0] }
+          ),
+          i18n.gettext('More add-ons by these authors'),
+          authorNames.length
+        );
+    }
+
+    const classnames = classNames('AddonDescription-more-addons', {
+      'AddonDescription-more-addons--theme': addonType === ADDON_TYPE_THEME,
```

Comment:
Do we want to support the `ADDON_TYPE_COMPLETE_THEME` here?

Reply:
No, we shouldn't show those I'd say.

Reply:
Oki, this is not the case anyway because the API call passes the `addonType` as filters.

Reply:
Yeah I was gonna say in general we don't tend to get complete themes unless a specific add-on is loaded, usually.